### DISCUSSION TITLE
Section 06 - PyTorch Custom Datasets

### DISCUSSION THREAD

**Discussion by leoproject** [mrdbourke/pytorch-deep-learning#367] [Ideas]
I would like material or suggestion about create my own dataset to object detection. How can I labeling images and organize files?

**Comment by longtongster** [comment]
Hi leoproject,

 Please find below a link to what I think is a great series on how to implement yolo1 or yolo3 from scratch youtube.

https://www.youtube.com/watch?v=t-phGBfPEZ4&list=PLhhyoLH6Ijfw0TpCTVTNk42NN08H6UvNq

I did this series about a year ago. Really step by step implemetation in pytorch. Also how to create the dataset

Have fun,

Kind Regards
The Longtongster

**Reply by leoproject** [reply]
Great, man, I'll look into it. I'll give you my feedback on it later.

**Reply by longtongster** [reply]
I hope you find the series in the link interesting
